app/components/MachinesPage/MachinesQuoteForm.tsx

```diff
@@ -0,0 +1,129 @@
+// LIBRARY IMPORTS
+import {
+  Button,
+  chakra,
+  FormControl,
+  Heading,
+  HStack,
+  Input,
+  Textarea,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { Dispatch, SetStateAction } from "react";
+import { useForm } from "react-hook-form";
+
+// LOCAL IMPORTS
+import { useSendQuoteRequest } from "@/utils/hooks/useSendQuoteRequest";
+import { PageViewMachinesBlocksQuoteSection } from "@/tina/__generated__/types";
+
+// TYPE DEFINTIONS
+interface MachineLayoutQuoteFormProps
+  extends PageViewMachinesBlocksQuoteSection {
+  setSubmitSuccessful: Dispatch<SetStateAction<boolean>>;
+}
+
+const ChakraForm = chakra("form");
+
+export default function MachinesQuoteForm(props: MachineLayoutQuoteFormProps) {
+  const { register, handleSubmit, errors, onSubmit } = useSendQuoteRequest();
+
+  return (
+    <ChakraForm
+      pt={14}
+      pb={24}
+      bg="brand.primary"
+      onSubmit={handleSubmit((data) =>
+        onSubmit(data, props.setSubmitSuccessful),
+      )}
+    >
+      <Heading textAlign="center" fontSize="3xl" mt={8} color="white">
```

Comment:
```suggestion
      <Heading textAlign="center" fontSize="3xl" fontWeight="normal" mt={8} color="white">
```
ChakraUI's default Heading style includes `fontWeight: bold`, but according to wireframe, it's normal.